doc/whatsnew/2.0.rst

```diff
@@ -16,6 +16,9 @@ New checkers
 * New ``useless-return`` message when function or method ends with a "return" or
   "return None" statement and this is the only return statement in the body.
 
+* New ``by-id-disabled-message`` message when a message is deactivated by id instead
```

Comment:
Let's add another short sentence explaining why using symbols is better (better mnemonic than simple ids, conveys the meaning right away etc)